04-Intro_to_Vue/assignment/vue-dub3/src/components/vue-Accordion.vue

```diff
@@ -0,0 +1,56 @@
+<script setup>
+import { ref } from 'vue'
+
+const name = ref('Accordion')
+</script>
+
+<template>
+  <h1>ACCORDION</h1>
+  <div class="container">
+              <div class="accordion" id="accordionExample">
```

Comment:
For these you want to loop through an array of data using v-for to display the accordions not having them hard coded like this